eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -3045,6 +3045,43 @@ groups:
           unit:     'GeV^-2'
   # }}}
 
+  # Parameters in D -> P Form Factor Parametrizations
+  # {{{
+  - title: 'Parameters in $B\to P$ Form Factor Parametrizations'
+    description: ''
+    parameters:
+      # form factor parameters for D->pi inspired by [BSZ:2015A]
+      'D->Pi::alpha^%1%_%2%@BSZ2015' :
+          central:  0.0
+          min:      0.0
+          max:      0.0
+          latex:    '$\alpha_{%1%,%2%}^{D \to \pi,\mathrm{BSZ2015}}$'
+          unit:     '1'
+          matrix:
```

Comment:
Ah, this is going to be a problem. One of the form factor parameters in ``f_0`` is idenfitied with an ``f_+`` parameter. So you create more parameters than needed. Which is bad, since an unknowing user might want to fit it and will not see it affected in a fit!

Reply:
Sure but this is treated internally, the spurious parameter will never been called, so the user can try to fit it, it just doesn't appear in the likelihood. I would keep it as it is now, unless you are strongly against. And this will be removed as soon as we provide default values for these parameters :)

Reply:
I like that solution!